app/src/main/java/com/github/houseorganizer/houseorganizer/billsharer/Billsharer.java

```diff
@@ -153,6 +200,19 @@ public void removeExpense(Expense expense) {
         updateExpenses();
```

Comment:
Shouldn't we refreshBalances after adding an expense too?

Reply:
No I do it when removing a debt but otherwise it will be refreshed when opening the balance tab

Reply:
Okey, just so I understand, why removing a debt yes and adding an expense no?